utils/vim/Makefile

```diff
@@ -8,20 +8,18 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vim
-PKG_VERSION:=9.1
+PKG_VERSION:=9.1.0821
+VIMVER=91
 PKG_RELEASE:=1
-VIMVER:=91
-
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
-PKG_SOURCE_URL:=https://www.vim.org/downloads
-PKG_HASH:=9c2d1d6763a672f6c5210a2bc4bad5a58e4eb8e8e3f293d950d9647811647ec7
 
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/vim/vim/tar.gz/refs/tags/v$(PKG_VERSION)?
+PKG_HASH:=808c14415391032873172bb6f6fbad1d211626ec475bd2d73dd5b91c3621f44d
```

Comment:
Can you please describe the reason, why we are now using GitHub in the commit description, please?

Reply:
For me, it looks like you are still using development builds. 🤔 
Based on:
- https://www.vim.org/download.php
- https://groups.google.com/g/vim_announce?pli=1

In the past, in the stable releases (OpenWrt), we were using stable versions, why do we need to use now development daily builds?